keylime/cloud_verifier_tornado.py

```diff
@@ -418,7 +429,17 @@ def post(self):
                     json_body = json.loads(self.request.body)
 
                     # Handle allowlists/IMA policies
+
+                    # How each pair of inputs should be handled:
+                    # - No name, no policy: use default empty policy
+                    # - Name, no policy: fetch existing policy from DB
+                    # - No name, policy: store policy using agent UUID as name
```

Comment:
Do we delete the policy after the agent is removed?

Reply:
@THS-on I was wondering the best way to go about this... there are some traps we could fall into.

For example, the user could add a policy named `list1` then add an agent using that new list. But if we delete the agent, we probably don't want also delete the policy, right? That list was added separately from the agent, and the user will expect it to stick around after deleting an agent that happened to use it.

But if we add a policy alongside the agent directly, without a name, the implication is likely that the user expects the policy to go away along with the agent when deleted.

The strategy I settled on, for now, is deleting the policy if it's named with the UUID of the agent that will be deleted. This will allow named policies to stick around while also deleting policies that were created without a name.